esp-backtrace/CHANGELOG.md

```diff
@@ -66,4 +72,4 @@ No changes - published to avoid conflicts with `esp-println`
 - Fix compilation for nightly after 2024-06-12. (#1681)
 - Only prints float registers on targets which have them. (#1690)
 
-[Unreleased]: https://github.com/esp-rs/esp-hal/commits/main/esp-backtrace?since=2025-01-15
+[Unreleased]: https://github.com/esp-rs/esp-hal/commits/main/esp-backtrace?since=2025-02-24
```

Comment:
We're no longer providing links for Unreleased in the currently open release PR, since this shows changes for _all_ packages since this date. Would you mind updating this to just link to the tagged release for this package, like we do in the release PR please?

Reply:
Done!